### ISSUE TITLE
Datastack archives don't package up data linked to in a CSV

### ISSUE THREAD

**Issue by phargogh** [natcap/invest#342]
We now have several models (HRA, CBC in the new refactor for sure, HQ in the `release/3.9` branch) that refer to other spatial files, either rasters or vectors, that are used in the model itself.  The datastack archive creation/extraction functions intended to create a self-contained archive of all data needed to run the model are missing these linked spatial files.

If we end up validating specific columns of data as spatial files (https://github.com/natcap/invest/issues/327), then maybe this new information in `ARGS_SPEC` could be used in datastack archiving as well?

**Comment by davemfish** [comment]
I'm adding a milestone to this one because I think it would be great to prioritize sometime soon-ish. In my opinion this is worth doing simply because it solves a problem we have on a regular basis: users share their data with us to debug something, but part of the datastack is missing. Back-and-forth ensues until we get all the data.

If we had a functional archive tool again, we could make it very easy to use in the workbench and it could become our default instruction when requesting people share their data.

**Comment by davemfish** [comment]
Also the ARGS_SPEC work is complete, so I think all the info to make this work is there.

**Comment by emlys** [comment]
This still might not be possible the HRA criteria scores table which isn't described in the args spec because it's too complicated.

**Comment by davemfish** [comment]
> This still might not be possible for the HRA criteria scores table which isn't described in the args spec because it's too complicated.

Oh good reminder. What if we just leave HRA behind on this one? Basically acknowledge the fact that the datastack archive will be incomplete for HRA, and then open a new issue for fixing that for HRA?

**Comment by phargogh** [comment]
I agree, let's do that.  I created issue #751 for this so we can tackle this for the other models.
